### PR TITLE
feat: Add anvil service to cli

### PR DESCRIPTION
**Pull request by roninjin10**
- Add anvil command to cli. This command spins up anvil and no other services

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* main
  * **PR #69** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/69" target="_blank">Graphite</a> 
    * **PR #70** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/70" target="_blank">Graphite</a> 
      * **PR #71** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/71" target="_blank">Graphite</a>  👈
        * **PR #74** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/74" target="_blank">Graphite</a> 
          * **PR #75** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/75" target="_blank">Graphite</a> 
            * **PR #76** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/76" target="_blank">Graphite</a> 
              * **PR #78** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/78" target="_blank">Graphite</a> 
                * **PR #80** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/80" target="_blank">Graphite</a> 
                  * **PR #81** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/81" target="_blank">Graphite</a> 
                    * **PR #82** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/82" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/71?utm_source=stack-comment).

File: cmd/cli.go [hunk 1]
```diff
@@ -23,6 +30,56 @@ func newCli(GitCommit string, GitDate string) *cli.App {
...
+					if err != nil {
+						log.Error("failed to load config", "err", err)
+						return err
+					}
+
+					processCtx, processCancel := context.WithCancel(ctx.Context)
+
+					runService := func(start func(ctx context.Context) error) {
+						wg.Add(1)
+						go func() {
```
Comment (hamdiallam) [inline cmd/cli.go:54]: If you're interested, @protolambda has worked on some lifecycle improvements for the indexer you can leverage here to manage resources more cleanly

https://github.com/ethereum-optimism/optimism/pull/7961

**Review by hamdiallam — APPROVED** [review]

File: cmd/cli.go (hunk 1 above)
Comment (roninjin10) [reply cmd/cli.go:54]: Definitely interested! Made a ticket https://github.com/ethereum-optimism/mocktimism/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=

File: cmd/cli.go [hunk 2]
```diff
@@ -23,6 +30,56 @@ func newCli(GitCommit string, GitDate string) *cli.App {
 				Description: "Display the current mocktimism config",
 				Action:      actionConfig,
 			},
+			{
+				Name:        "anvil",
```
Comment (roninjin10) [inline cmd/cli.go:34]: Note I'm adding individual service commands during development but the actual release will not have these. These are here mostly to help with development now

**Pull request merged**